community/berry/version

```diff
@@ -1 +1 @@
-0.1.7 1 
+0.1.8 2 
```

Comment:
This should just be `0.1.8 1`

Reply:
Ah, that's me misunderstanding the docs! What does this phrase mean?

> the second field should contain the version number of the repository files themselves

I took it to mean that every time the repository files get updated to a new package version, that number is incremented.

Reply:
I see how that can be confusing. 
You can think of the second version field as being the version of those files *at that version*. Some people call this an epoch (or less commonly, age) version. Essentially, this version changes when the package itself has changed in some way that requires a *rebuild*, even though the package itself has not changed (see the [recent changes to alsa-lib](https://github.com/kiss-community/repo/commit/83fde325c2f710dbe933e8b6cb2c74c11ec0a127) for an example).